plugin/webpack.config.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+
+const webpack = require( 'webpack' );
+
+const config = module.exports = {
+	entry: {
+		app: './src/index.js'
```

Comment:
What about the distinction `src`/`build`? Do you think we should use `editor` as a parent directory or it's ok to keep `build` as is?